pkg/nuclide-diagnostics-ui/lib/main.js

```diff
@@ -135,18 +135,19 @@ class Activation {
         .switchMap(updater => (
           updater == null ? Observable.of([]) : updater.allMessageUpdates
         )),
-      this._state.filterByActiveTextEditor,
-      // https://github.com/gajus/eslint-plugin-flowtype/pull/131
-      // eslint-disable-next-line flowtype/space-after-type-colon
-      (featureConfig.observeAsStream('nuclide-diagnostics-ui.showDiagnosticTraces'):
-          Observable<any>),
+      // eslint-disable-next-line max-len
```

Comment:
Can you be more specific as to how you'd like this broken up on multiple lines (not sure of the best practices for splitting casting like this) or could you just edit it according to your preference before you accept (I'll take note for future pull requests)?

https://github.com/blog/1738-web-flow-editing-from-pull-requests